tests/acceptance/pageObjects/FilesPageElement/filesList.js

```diff
@@ -36,6 +36,46 @@ module.exports = {
           }
         })
     },
+
+    /**
+     * Checks whether a given action is disabled for a given file name.
+     * This method does find out itself if the file-action burger has to be clicked or not.
+     *
+     * @param {string} fileName
+     * @param {string} delete|share|rename
+     * @param {function(boolean)} callback - whether the action is disabled
+     * @returns {*}
+     */
+    isActionDisabled: function (fileName, action, callback) {
+      const btnSelectorHighResolution = '(' + this.getFileRowSelectorByFileName(fileName) +
+        this.elements[action + 'ButtonInFileRow'].selector + ')[1]'
+      const btnSelectorLowResolution = '(' + this.getFileRowSelectorByFileName(fileName) +
+        this.elements[action + 'ButtonInFileRow'].selector + ')[last()]'
+      const fileActionsBtnSelector = this.getFileRowSelectorByFileName(fileName) +
+        this.elements.fileActionsButtonInFileRow.selector
```

Comment:
put in separate function to generate this selectors, so it also used for https://github.com/owncloud/phoenix/pull/1951/commits/195d9b18222db8618d6ee5176a7989346f07e522#diff-2da1b8066c5b1ccfa1ef0555b24164dbL16

Reply:
done